config-example/docker/myhost/.env

```
@@ -69,6 +69,10 @@ CROWDSEC_BOUNCER_API_KEY="use-some-very-secure-value-here"
 # Then create the user on Settings -> Admins
 UNIFI_LOCAL_VIEWONLY_USERNAME="use-some-very-secure-value-here"
 UNIFI_LOCAL_VIEWONLY_PASSWORD="use-some-very-secure-value-here"
+# Used to connect the shared mongodb instance
+UNIFI_MONGO_DBNAME="unifi"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)